lib/api/core/httpRouter/index.js

```diff
@@ -136,6 +136,10 @@ class Router {
       return replyWithError(cb, request, new BadRequestError(`Unrecognized HTTP method ${httpRequest.method}`));
     }
 
+    if (httpRequest.url.length > 1 && httpRequest.url.endsWith('/')) {
```

Comment:
This code only solves the problem if a single trailing slash is added at the end of an URI. 
Instead, it should trim any number of trailing slashes from it.